<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetLocate</class>
 <widget class="QDialog" name="SetLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置雷达坐标</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {    
	background: white url(:/images/jpg/sansi_logo.jpg);
    background-repeat: repeat-y;
    background-position: left;
}

QLineEdit { 
	min-width:68px;
    min-height:28px;
	background-color: rgba(0, 255, 255, 150);
	border-color: rgb(255, 255, 127);
	font: 14pt &quot;Myanmar Text&quot;;
	
}
QFrame {	
	background-color: rgba(0, 75, 125, 50);
}

QLabel{
	width: 100px;
	height: 100px;
	border: 2px solid white;
	border-radius: 4px;
	font: 75 12pt &quot;Microsoft JhengHei UI&quot; ;
	text-align: center;
}

QPushButton{
	background-color: rgb(255, 255, 255);
    min-width:68px;
    min-height:28px;
	border-radius: 10px;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No.6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No.4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No.3</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>lineEdit_1</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>setButton</zorder>
  <zorder>quitButton</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>lineEdit_8</zorder>
  <zorder>lineEdit_9</zorder>
  <zorder>lineEdit_10</zorder>
  <zorder>lineEdit_11</zorder>
  <zorder>lineEdit_12</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>SetLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
